sdk/iot/provisioning/samples/src/paho_iot_provisioning_example.c

```diff
@@ -262,6 +220,119 @@ static int register_device()
   return 0;
 }
 
+static void print_buffer(az_span span)
+{
+  char* buffer = (char*)az_span_ptr(span);
+  for (int i = az_span_size(span); i > 0; i--)
+  {
+    putchar(*buffer++);
+  }
+
+  putchar('\n');
+}
+
+static int get_operation_status()
+{
+  int rc;
+  bool registration_complete = false;
+  char* topicName;
+  int topicLen;
+  MQTTClient_message* message;
+
+  while (!registration_complete)
+  {
+    if ((rc = MQTTClient_receive(mqtt_client, &topicName, &topicLen, &message, 60 * 1000))
```

Comment:
>60 * 1000 [](start = 79, length = 9)

is this timeout? Can you please declare some macro to associate context with magic number?